include/item_menu_icons.h

```diff
@@ -1,11 +1,11 @@
 #ifndef GUARD_ITEM_MENU_ICONS
 #define GUARD_ITEM_MENU_ICONS
 
-extern const struct CompressedSpriteSheet gUnknown_0857FB34;
-extern const struct CompressedSpriteSheet gUnknown_0857FB3C;
-extern const struct CompressedSpritePalette gUnknown_0857FB44;
-extern const struct CompressedSpriteSheet gUnknown_0857FDEC;
-extern const struct CompressedSpritePalette gUnknown_0857FDF4;
+extern const struct CompressedSpriteSheet gBagMaleTable;
```

Comment:
These should be named more like `gBagMaleSpriteSheet`, rather than the `Table` suffix.